cves/CVE-2015-1831.yml

```diff
@@ -207,4 +209,4 @@ mistakes:
     Use those questions to inspire your answer. Don't feel obligated to answer
     every one. Write a thoughtful entry here that those in the software
     engineering industry would find interesting.
-  answer:
```

Comment:
I need more here. How could it have been missed?

Reply:
Added more to better explain thoughts.